### DISCUSSION TITLE
6 Project Ideas (very rough ideas)

### DISCUSSION THREAD

**Discussion by jakegros** [USU-ITLS-x265-23-summer/class-repo#63] [Ideas]
These are some ideas I'm interested in for projects. They are extremely vague at this point and I'm not sure yet what is beyond the scope of this class:
--I'd like to create the platform of my portfolio, if that's a thing. Instead of using this course to create something to put in my portfolio, I would like to build a site that would hold my portfolio 
--website of a topic I'm passionate about   -Mindfulness, birds, accessible apps, poetry 
--create simple games / archive 
--create accessible training documentation 
--something with AI integration 
--something utilizing gamification with my student employees

**Comment by metasean** [comment]
- platform of my portfolio
  - Something like this doesn't have a fancy, generic name like "blog", but people definitely do build platforms for their portfolios

- passion website
  - always a safe bet

- simple games / archive
  - as in multiple games or an archive of games?

- accessible training documentation
  - I'm a sucker for anything that might improve accessibility
  - fwiw, [Week 6](https://github.com/USU-ITLS-x265-23-summer/class-repo/blob/main/Weekly-assignments/Week-06.md) is an intro to some very basic accessibility concerns for web development

- AI integration
  - this would definitely take you beyond what we're explicitly covering in this course, but you're in the Master's variation, so stepping slightly beyond is one of the requirements for your project.

- gamification with my student employees
  - definitely on the "extremely vague" side, but it's exactly the type of item I was looking for both in it's specificity and vagueness 😁


I'm excited to see which of these you end up completing!

**Comment by derekscholes** [comment]
I actually like the last idea. Something different than the norm. Definitely something I would be interested in seeing and maybe slightly more intense than what I thought of.
